test/compute_kernels/ldpc/test_ldpc_mod.cpp

```diff
@@ -188,6 +188,13 @@ int main(int argc, char* argv[]) {
       bblib_ldpc_decoder_5gnr(&ldpc_decoder_5gnr_request,
                               &ldpc_decoder_5gnr_response);
     }
+
+    // Descramble the deocded buffer
+    for (size_t i = 0; i < num_codeblocks; i++) {
+      WlanScramble((int8_t*)decoded_codewords[i], ldpc_config.NumCbLen() / 8,
```

Comment:
c++ style cast.  However,  if we are casting this a whole bunch, does this even make sense?  Ie should we just have a function that takes a uint8_t pointer?

Reply:
Let's discuss it in this afternoon meeting.